upload-sourcemaps.js

```diff
@@ -81,6 +81,35 @@ module.exports = async (options) => {
 
     output = uploadResult.stdout.toString();
     log(output);
+
+    setCommits = setCommits || process.env.SENTRY_EXPO_SET_COMMITS;
+    if (setCommits) {
+      let commitsResult = await spawnAsync(
+        sentryCliBinaryPath,
+        ['releases', 'set-commits', '--auto', version],
+        {
+          env: childProcessEnv,
+        }
+      );
+
+      output = commitsResult.stdout.toString();
+      log(output);
+    }
+
+    deployEnv = deployEnv || process.env.SENTRY_EXPO_DEPLOY_ENV;
+    if (deployEnv) {
```

Comment:
```suggestion
    if (deployEnv || process.env.SENTRY_EXPO_DEPLOY_ENV) {
```

Reply:
My thought in using the var here was to reuse it for the env name... otherwise we'd have to also do `['releases', 'deploys', version, 'new', '-e', deployEnv || process.env.SENTRY_EXPO_DEPLOY_ENV]` right?

Reply:
ah yeah, you're right, ignore this suggestion